backend/app/assets/javascripts/backend/forms.coffee

```diff
@@ -0,0 +1,11 @@
+jQuery ->
```

Comment:
Let's please use js, and remove CoffeeScript from the project. Our front end developers use plain javascript syntax, let's avoid adding too many things to the mixture.